README.md

```diff
@@ -1,2 +1,28 @@
 # github_practice_2023_2
-Practice Session
+
+For your assignment, answer the following questions
+
+1. What is git and github? (Bernice)
+
+
+2. What is a repository?(Lovis)
+
+
+3. What is a branch? (Ted)
+
```

Comment:
it is a source which allows you merge feature ideas  in a contained area of repository .